src/db/migrations/20190403212306-create-article.js

```diff
@@ -34,6 +34,10 @@ export default {
         type: Sequelize.STRING,
         allowNull: false,
       },
+      tags: {
+        type: Sequelize.ARRAY(Sequelize.STRING),
+        allowNull: true,
```

Comment:
You have written correct code. A suggestion I can give is that, by default, allowNull is already set to true, so you do not have to explicitly write it here, if true is the condition you want.

Reply:
Ok,